7_react/react1/week2/src/compoents/TaskList.jsx

```diff
@@ -0,0 +1,37 @@
+import Checkbox from "./Checkbox";
+
+const TaskList = props => {
+  const { list, setList } = props;
+  const onChangeStatus = element => {
+    const { name, checked } = element.target;
+
+    const updateList = list.map(item => ({
+      ...item,
+      done: item.id === name ? checked : item.done
+    }));
+    setList(updateList);
+  };
+
+  const onClickRemoveItem = () => {
+    const updateList = list.filter(item => !item.done);
+    setList(updateList);
+  };
+
+  const checkList = list.map(item => (
+    <Checkbox key={item.id} data={item} onChange={onChangeStatus} />
+  ));
+  return (
+    <div className="todoList">
+      {list.length ? checkList : "Add your tasks"}
```

Comment:
Try combining these two lines to use `{list.length ? ...}` once. 
You can use the short syntax for a fragment to combine the components: https://legacy.reactjs.org/docs/fragments.html#short-syntax